### PR TITLE
drakrun: adjustments to cfg.template

### LINKED ISSUE #449: Out of memory in draksetup
**Describe the bug**
```
Parsing config from /etc/drakrun/configs/vm-0.cfg
xc: error: panic: xc_dom_boot.c:122: xc_dom_boot_mem_init: can't allocate low memory for domain: Out of memory
libxl: error: libxl_dom.c:593:libxl__build_dom: xc_dom_boot_mem_init failed: Cannot allocate memory
libxl: error: libxl_create.c:1576:domcreate_rebuild_done: Domain 1:cannot (re-)build domain: -3
libxl: error: libxl_domain.c:1182:libxl__destroy_domid: Domain 1:Non-existant domain
libxl: error: libxl_domain.c:1136:domain_destroy_callback: Domain 1:Unable to destroy guest
libxl: error: libxl_domain.c:1063:domain_destroy_cb: Domain 1:Destruction of domain failed
```

**How to reproduce**

1. Install drakcore and drakrun on vmware debian buster machine
2. Configurations of the VM
- 5172MB ram
- 1 processor, 8 cores
- xfce DE is installed in the VM
2. Execute `draksetup install winall.iso`

### PR DESCRIPTION
**Pull request by icedevml**
resolve #449

### CODE REVIEW

**Comment by icedevml** [comment]
So right now we need better `install_info.json` on the CI, I suppose.

File: drakrun/drakrun/scripts/cfg.template [hunk 1]
```diff
@@ -24,4 +29,4 @@ cpuid="host,htt=0"
 vga="stdvga"
 vif = [ 'type=ioemu,model=e1000,bridge=drak{{ VM_ID }}' ]
 disk = [ {{ DISKS }} ]
-processor_trace_buf_kb = 65536
+processor_trace_buf_kb = 8192
```
Comment (manorit2001) [inline drakrun/drakrun/scripts/cfg.template:32]: Just curious, what does this value do?

Comment (icedevml) [reply drakrun/drakrun/scripts/cfg.template:32]: This is reserved for future - Intel Processor Trace support. Right now it's a no-op until you have approproate Xen version.

Comment (manorit2001) [reply drakrun/drakrun/scripts/cfg.template:32]: Ah, okay thanks, will read more about `Intel Processor Trace support`

File: drakrun/drakrun/config.py [hunk 2]
```diff
@@ -16,6 +16,8 @@
 
 @dataclass
 class InstallInfo(DataClassJsonMixin):
+    vcpus: int
+    memory: int
```
Comment (chivay) [inline drakrun/drakrun/config.py:20]:
```suggestion
    vcpus: int = 2
    memory: int = 3048
```

File: drakrun/drakrun/scripts/cfg.template [hunk 3]
```diff
@@ -1,11 +1,16 @@
...
-memory = 3048
-vcpus = 2
-maxvcpus = 2
+maxmem = {{ MEMORY }}
+memory = {{ MEMORY }}
+vcpus = {{ VCPUS }}
+maxvcpus = {{ VCPUS }}
 type = "hvm"
-boot = "cd,menu=on,splash=/usr/share/drakrun/splash.jpg,splash-time=2000"
+boot = "cd"
```
Comment (chivay) [inline drakrun/drakrun/scripts/cfg.template:13]: If we're removing the splash screen, we should also delete it from repository and deb scripts

Comment (icedevml) [reply drakrun/drakrun/scripts/cfg.template:13]: Unfortunately it never really worked due to some complicated Xen/qemu related issues :F As this is a deadcode, I want to remove it.

:+1: I will also remove from the repository and deb scripts.

File: drakrun/drakrun/scripts/cfg.template (hunk 1 above)
Comment (chivay) [inline drakrun/drakrun/scripts/cfg.template:32]:
```suggestion
vmtrace_buf_kb = 8192
```

**Review by chivay — APPROVED** [review]

File: drakrun/drakrun/scripts/cfg.template (hunk 1 above)
Comment (icedevml) [reply drakrun/drakrun/scripts/cfg.template:32]: 🙉

**Pull request merged**